common/src/main/java/org/terracotta/angela/common/distribution/Distribution107InlineController.java

```diff
@@ -178,11 +181,10 @@ private synchronized Object startIsolatedServer(Path kitDir, String serverName,
       URL url = tc.toUri().toURL();
       URL resource = serverWorking.toUri().toURL();
       System.setProperty("tc.install-root", kitDir.resolve("server").toString());
-      System.setProperty("restart.inline", "true");
 
       ClassLoader loader = new IsolatedClassLoader(new URL[] {resource, url});
-      Method m = Class.forName("com.tc.server.TCServerMain", true, loader).getMethod("createServer", List.class);
-      return m.invoke(null, cmd);
+      Method m = Class.forName("com.tc.server.TCServerMain", true, loader).getMethod("createServer", List.class, OutputStream.class);
+      return m.invoke(null, cmd, Files.newOutputStream(serverWorking.resolve("stdout.txt"), StandardOpenOption.CREATE, StandardOpenOption.APPEND));
```

Comment:
How the stream is closed ? Will it be closed even when there is an exception thrown at startup from DC or when server is shutdown because of an exception in the passive sync, or a request for termination (stop) ?

Reply:
The stream is passed to logback and that code assumes control of the stream. You have a good point that some best effort should be made to close the stream on server shutdown. I'll check that code in core.